auth/realm/base/src/main/java/org/wildfly/security/auth/realm/JaasSecurityRealm.java

```diff
@@ -148,54 +148,10 @@ public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evi
         return SupportLevel.POSSIBLY_SUPPORTED;
     }
 
-    /**
-     * @param entry           login configuration file entry
-     * @param subject         classLoader to use with LoginContext, this class loader must contain LoginModule CallbackHandler classes
-     * @param callbackHandler callbackHandler to pass to LoginContext
-     * @return the instance of LoginContext
-     * @throws RealmUnavailableException
-     */
-    private LoginContext createLoginContext(final String entry, final Subject subject, final CallbackHandler callbackHandler) throws RealmUnavailableException {
-        if (jaasConfigFilePath != null) {
-            File file = new File(this.jaasConfigFilePath);
-            if (!file.exists() && !file.isDirectory()) {
-                throw ElytronMessages.log.failedToLoadJaasConfigFile();
-            }
-        }
-        try {
-            if (jaasConfigFilePath == null) {
-                return new LoginContext(entry, subject, callbackHandler);
-            } else {
-                return new LoginContext(entry, subject, callbackHandler, Configuration.getInstance(DEFAULT_CONFIGURATION_POLICY_TYPE, new URIParameter(jaasConfigFilePath)));
-            }
-        } catch (LoginException | NoSuchAlgorithmException le) {
-            throw ElytronMessages.log.failedToCreateLoginContext(le);
-        }
-    }
-
-    private CallbackHandler createCallbackHandler(final Principal principal, final Evidence evidence) {
-        if (handler != null) {
-            try {
-                final CallbackHandler callbackHandler = handler.getClass().getConstructor().newInstance();
-                // custom handlers were allowed in the past as long as they had a public setSecurityInfo method. Use this method if it exists
-                final Method setSecurityInfo = handler.getClass().getMethod("setSecurityInfo", Principal.class, Object.class);
-                setSecurityInfo.invoke(callbackHandler, principal, evidence);
-                return callbackHandler;
-            } catch (InstantiationException | IllegalAccessException | NoSuchMethodException | InvocationTargetException e) {
-                // ignore if this method does not exist
-                return handler;
-            }
-        } else if (Security.getProperty("auth.login.defaultCallbackHandler") != null) {
-            // security property "auth.login.defaultCallbackHandler" is not null so LoginContext will initialize it itself
-            return null;
-        } else {
-            return new JaasSecurityRealmDefaultCallbackHandler(principal, evidence);
-        }
-    }
-
     private class JaasRealmIdentity implements RealmIdentity {
 
         private final Principal principal;
+        private URI jaasConfigFilePath;
```

Comment:
@Kshitiz-Mhto Thank you for the PR! Just a minor, this should not be added here, rather this class should use `jaasConfigFilePath` from its parent. You can try running `mvn clean install` locally to see if your changes pass.

Reply:
@Kshitiz-Mhto Sorry I meant to say this class should use `jaasConfigFilePath` from its outer class, not parent

Reply:
Hi @Skyllarr, when i removed the line 154 as per your suggestion, it leads to a error in line  

![line264](https://user-images.githubusercontent.com/102871719/188931746-6ecc88d0-b98d-4e38-963a-15306dd083de.png)

which says `cannot resolve symbol `

---
then when i refactor the line 264 from `this.jaasConfigFilePath` ---> `jaasConfigFilePath` , error gone

Reply:
@Kshitiz-Mhto Yes, changing `this.jaasConfigFilePath` to `jaasConfigFilepath` is a good fix! 

Feel free to update your PR with this change. If you encounter any problems with git, just let us know.

Reply:
@Skyaller thank you for your warm welcome and for the opporunity. Another thing, i m having some issue with building this project in my intellij, should i post those error screenshot on  zulip?

Reply:
@Kshitiz-Mhto Sure! Yes you can post those here or on Zulip, @fjuma or @cam-rod might be able to help you. 

Just some general tips for building, you should use Java 11 and have JAVA_HOME variable configured.

Reply:
Hey @Kshitiz-Mhto, we've just added some configuration guides for Elytron, [take a look](https://wildfly-security.github.io/wildfly-elytron/guides) and let me know if you have any issues.

Reply:
Successfully  executed `sudo mvn clean install` thank you :+1: